src/EnergyPlus/DataDaylighting.hh

```diff
@@ -57,6 +57,7 @@
 // EnergyPlus Headers
 #include <EnergyPlus/Data/BaseData.hh>
 #include <EnergyPlus/DataGlobals.hh>
+#include <EnergyPlus/DataSurfaces.hh>
```

Comment:
Need this for the `FWC` enumeration. May move that enumeration later to avoid this.